examples/datetime-flux/js/client.js

```diff
@@ -1,7 +1,7 @@
-var Iso = require('../../../')
-var React = require('react')
-var AltIsomorphicElement = require('../src/components/AltIsomorphicElement')
+let Iso = require('../../../')
```

Comment:
yeah it was compiled with babel, can we use the es6 imports for this example then, I was wondering about that?